src/frontend/env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="vite/client" />
+declare module "envConfig";
```

Comment:
What is this for?

Reply:
> TypeScript, by default, does not recognize static asset imports as valid modules. To fix this, include [vite/client](https://vitejs.dev/guide/features.html#client-types).

https://vitejs.dev/guide/assets.html

Reply:
Can you add a note about this in the file?

Reply:
Added a link to the doc